packages/analytics-core/src/timeline.ts

```diff
@@ -74,15 +74,27 @@ export class Timeline {
     );
 
     for (const plugin of before) {
-      event = await plugin.execute({ ...event });
+      const e = await plugin.execute({ ...event });
+      if (e == null) {
```

Comment:
Would prefer threequals, which I should add to eslint rules to better enforce 😄 

```suggestion
      if (e === null) {
```

Reply:
Fixed